### ISSUE TITLE
ModuleNotFoundError: No module named 'retry'

### ISSUE THREAD

**Issue by linho1150** [RavenPack/python-api#15]
Hi, The following error occurred while installing the recently updated version 1.1.1. 
The error does not occur in 1.1.0 and appears to be a package installation related issue.

Thank you :)

```
38.84   Preparing metadata (setup.py): started
38.99   Preparing metadata (setup.py): finished with status 'done'
39.04 Collecting ravenpackapi (from ***==1.0.1)
39.08   Downloading ravenpackapi-1.1.1.tar.gz (46 kB)
39.09      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 46.3/46.3 kB 3.9 MB/s eta 0:00:00
39.10   Preparing metadata (setup.py): started
39.23   Preparing metadata (setup.py): finished with status 'error'
39.23   error: subprocess-exited-with-error
39.23   
39.23   × python setup.py egg_info did not run successfully.
39.23   │ exit code: 1
39.23   ╰─> [16 lines of output]
39.23       Traceback (most recent call last):
39.23         File "<string>", line 2, in <module>
39.23         File "<pip-setuptools-caller>", line 34, in <module>
39.23         File "/tmp/pip-install-8fjxf8li/ravenpackapi_bd28336da7f245fcb0e75504dfa51640/setup.py", line 3, in <module>
39.23           from ravenpackapi import __version__
39.23         File "/tmp/pip-install-8fjxf8li/ravenpackapi_bd28336da7f245fcb0e75504dfa51640/ravenpackapi/__init__.py", line 2, in <module>
39.23           from .core import RPApi
39.23         File "/tmp/pip-install-8fjxf8li/ravenpackapi_bd28336da7f245fcb0e75504dfa51640/ravenpackapi/core.py", line 14, in <module>
39.23           from ravenpackapi.upload.module import UploadApi
39.23         File "/tmp/pip-install-8fjxf8li/ravenpackapi_bd28336da7f245fcb0e75504dfa51640/ravenpackapi/upload/module.py", line 7, in <module>
39.23           from ravenpackapi.upload.models import FILE_FIELDS, FOLDER_FIELDS, File, Folder
39.23         File "/tmp/pip-install-8fjxf8li/ravenpackapi_bd28336da7f245fcb0e75504dfa51640/ravenpackapi/upload/models.py", line 4, in <module>
39.23           from ravenpackapi.upload.upload_utils import retry_on_too_early
39.23         File "/tmp/pip-install-8fjxf8li/ravenpackapi_bd28336da7f245fcb0e75504dfa51640/ravenpackapi/upload/upload_utils.py", line 3, in <module>
39.23           from retry.api import retry_call
39.23       ModuleNotFoundError: No module named 'retry'
39.23       [end of output]
39.23   
39.23   note: This error originates from a subprocess, and is likely not a problem with pip.
39.24 error: metadata-generation-failed
39.24 
39.24 × Encountered error while generating package metadata.
39.24 ╰─> See above for output.
39.24 
39.24 note: This is an issue with the package mentioned above, not pip.
39.24 hint: See above for details.
```

In additional, my computer already install requests package. if i installed this package in new python venv, i see another error. I already tried update pip, update setuptools but It couldn't fix it. Is it setuptools error?

```
Collecting ravenpackapi
  Using cached ravenpackapi-1.1.1.tar.gz (46 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

...
      ModuleNotFoundError: No module named 'requests'
      [end of output]
...
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by joserc87** [comment]
Thank you for reporting! Indeed this is an issue with the latest version and I'm currently releasing a new version with a hotfix. Sorry for the inconveniences. :bow:

**Comment by joserc87** [comment]
@Linho1150 the issue should be fixed in [v1.1.2](https://github.com/RavenPack/python-api/releases/tag/v1.1.2). I'll leave this issue open for a couple of days in case you want to confirm that it works on your side

**Comment by linho1150** [comment]
@joserc87 Thanks for the quick fix! 👏👏
